### DISCUSSION TITLE
Set the first letter of the month in uppercase and remove the year

### DISCUSSION THREAD

**Discussion by marcogian2001** [hi-manshu/Kalendar#36] [Q&A]
Hi I'm here to ask if there is a way to set the first letter of the month in uppercase and remove the year in the Oceanic Kalendar. Thanks

**Comment by gsrathoreniks** [comment]
Month name is as per your need but right it's followed by the Year, for ex. "August 2022".
Will try to provide a way to config the Month-Header some time soon.
